tests/ManageCourses.Tests/DbIntegration/DataServiceTests.cs

```diff
@@ -435,7 +455,8 @@ private static UcasPayload GetUcasCoursesPayload()
                     new UcasCourse
                     {
                         InstCode = InstCode1,
-                        CrseCode = "CourseCode_1"
+                        CrseCode = "CourseCode_1",
+                        Status = "N"
```

Comment:
you can see in this the effect on the diff of not previously having the trailing comma, it turns what would have been a one-line addition into a two line addition and one line deletion

Reply:
Done